tests/unit/test-waiter-test.ts

```diff
@@ -1,22 +1,46 @@
+// @ts-ignore
 import { module, test } from 'qunit';
 import { Promise } from 'rsvp';
 import { TestWaiter, _reset, getWaiters, getPendingWaiterState } from 'ember-test-waiters';
 import MockStableError, { overrideError, resetError } from './utils/mock-stable-error';
 
-module('test-waiter', function(hooks) {
+module('test-waiter', function(hooks: NestedHooks) {
   hooks.afterEach(function() {
     _reset();
     resetError();
   });
 
-  test('test waiter can be instantiated with a name', function(assert) {
+  test('test waiter can be instantiated with a name', function(assert: Assert) {
```

Comment:
Uhh, why do you have to type this? Seems related to the ts-ignore at the top, we should absolutely fix...

Reply:
Correct. Hopefully it's resolved once the fix for `@types/qunit` is resolved.

Reply:
I need to get around to a fix for [the `ember-qunit` types](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/ember-qunit). @rwjblue pointed out that any "ES6 module-ification" of QUnit belongs there